### PR TITLE
Changes

### PR DESCRIPTION
**Pull request by pmuir**

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 4134f5a27173901cab6d6b612912fd087799e3b2
  * Your subject line is longer than 50 characters

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by pmuir** [comment]
DO NOT MERGE YET

**Comment by fabric8cd** [comment]
@pmuir snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1062-fabric8-ui.int.rdu2c.fabric8.io

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 4134f5a27173901cab6d6b612912fd087799e3b2
  * Your subject line is longer than 50 characters

 * Commit: 31c15e8bf6b934d5bff3e26b850e062fda211a8b
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by fabric8cd** [comment]
@pmuir snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-1062-fabric8-ui.int.rdu2c.fabric8.io

File: src/app/create/pipelines/pipelines.component.html [hunk 1]
```diff
@@ -1,5 +1,14 @@
 <div id="pipelines" class="content container">
-  <alm-toolbar [config]="toolbarConfig" (onFilterChange)="filterChange($event)" (onSortChange)="sortChange($event)"></alm-toolbar>
-  <hr />
+  <alm-toolbar [config]="toolbarConfig"
+               [viewsTemplate]="addOpenShiftConsoleLink"
+               (onFilterChange)="filterChange($event)"
+               (onSortChange)="sortChange($event)">
+    <ng-template #addOpenShiftConsoleLink>
```
Comment (joshuawilson) [inline src/app/create/pipelines/pipelines.component.html]: This looks wrong but I want to move this forward. @dlabrecq can you please submit an issue with directions on this, please.

**Pull request merged**